ckan/lib/webassets_tools.py

```diff
@@ -40,13 +49,23 @@ def create_library(name: str, path: str) -> None:
     # TODO: make PR into webassets with preferable solution
     # Issue: https://github.com/miracle2k/webassets/issues/519
     for name, bundle in bundles.items():
-        env._named_bundles.pop(name, None)
-        env.register(name, bundle)
+        try:
```

Comment:
Note: This is silently overriding webassets if they are duplicated. Instead of silently doing it I added a warning. Testing it with 10+ extensions the only one going into this issue is `ckanext-geoview` wich is overriding due to (what i think) it is an issue in a duplicated call in subclasses: https://github.com/ckan/ckanext-geoview/blob/97036fabd85d2a6f80a2d6b3bf48414a03dcb825/ckanext/geoview/plugin/__init__.py#L176